src/checker.ml

```diff
@@ -965,6 +965,36 @@ let view_current_liquidation_auction_minimum_bid ((), state: unit * checker) : v
   ; minimum_bid = liquidation_auction_current_auction_minimum_bid auction
   }
 
+let view_current_liquidation_auction_id ((), state: unit * checker) : liquidation_auction_id option =
+  assert_checker_invariants state;
+  match state.liquidation_auctions.current_auction with
+  | None -> None
+  | Some current_auction -> Some current_auction.contents
+
+let view_current_liquidation_auction_winning_bid ((), state: unit * checker) : bid option =
+  assert_checker_invariants state;
+  let current_auction = liquidation_auction_get_current_auction state.liquidation_auctions in
+  match current_auction.state with
+  | Descending _ -> (None: bid option)
+  | Ascending (bid, _, _) -> Some bid
```

Comment:
`liquidation_auction_get_current_auction` already fails if there is pending auction so I think you could make all these views fail if there is no ongoing auction; no need to deal with `option` types :wink: